src/pages/test-report/components/vehicle-check/vehicle-check.ts

```diff
@@ -21,12 +21,13 @@ import { isSeriousMode, isDangerousMode, isRemoveFaultMode } from '../../test-re
 import { map } from 'rxjs/operators';
 import { merge } from 'rxjs/observable/merge';
 import { isEmpty } from 'lodash';
+import { Observable } from 'rxjs/Observable';
 
 @Component({
   selector: 'vehicle-check',
   templateUrl: 'vehicle-check.html',
 })
-export class VehicleCheckComponent implements OnInit {
+export class VehicleCheckComponent implements OnInit, OnDestroy {
```

Comment:
Then declaration of `OnDestroy` has been added here - inconsistent with the above comment.

Reply:
Update the above. Now consistent with this class.